test/sharness/t0113-gateway-symlink.sh

```diff
@@ -9,26 +9,22 @@ test_description="Test symlink support on the HTTP gateway"
 test_init_ipfs
 test_launch_ipfs_daemon
 
-
-test_expect_success "Create a test directory with symlinks" '
-  mkdir testfiles &&
-  echo "content" > testfiles/foo &&
-  ln -s foo testfiles/bar &&
-  test_cmp testfiles/foo testfiles/bar
-'
-
-test_expect_success "Add the test directory" '
-  HASH=$(ipfs add -Qr testfiles)
+# Import test case
+# See the static fixtures in ./t0113-gateway-symlink/
+test_expect_success "Add the test directory with symlinks" '
+  ipfs dag import ../t0113-gateway-symlink/testfiles.car
 '
+ROOT_DIR_CID=QmWvY6FaqFMS89YAQ9NAPjVP4WZKA1qbHbicc9HeSKQTgt # ./testfiles/
 
 test_expect_success "Test the directory listing" '
-  curl "$GWAY_ADDR/ipfs/$HASH/" > list_response &&
+  curl "$GWAY_ADDR/ipfs/$ROOT_DIR_CID/" > list_response &&
   test_should_contain ">foo<" list_response &&
   test_should_contain ">bar<" list_response
 '
 
+# TODO: it's unclear to me if that `echo -n "foo" > expected` should be a fixture somehow.
```

Comment:
I don't think it has to be a fixture, as it is the same file. We only store it to a file because `test_cmp` requires two files for comparison I think.

Reply:
removed, thanks for the confirmation!